src/Evolution/Systems/Cce/Equations.hpp

```diff
@@ -467,6 +468,12 @@ struct ComputeBondiIntegrandFromVariables {
   }
 
  private:
+  // gcc believes temporary_cache is unused, but that is incorrect, it is
+  // forwarded in the parameter pack expansion.
+#ifndef __clang__
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-but-set-parameter"
+#endif  // __clang__
```

Comment:
I don't know if this is the best-practice way to do this, so perhaps another reviewer can confirm

Reply:
Do you mean specifically the `#ifndef __clang__` to select compiler? This was the method I was advised to use for a compiler-specific warning in #1145 , though the warning was ultimately removed by other changes, but I'm happy to entertain other methods.